apps/src/sites/studio/pages/code-studio.js

```diff
@@ -105,3 +108,12 @@ window.CDOSounds = Sounds.getSingleton();
 
 checkForUnsupportedBrowsersOnLoad();
 initHamburger();
+
+$(document).ready(() => {
+  if (window.userNameCookieKey) {
+    const val = cookies.get(window.cookieKey);
+    if (val) {
+      store.dispatch(setUserSignedIn(true));
```

Comment:
Should probably always dispatch `setUserSignedIn`, but with `false` if we don't have our cookie.